cms/blocks.py

```diff
@@ -55,3 +57,36 @@ class FacultyBlock(blocks.StructBlock):
     description = blocks.RichTextBlock(
         help_text="A brief description about the faculty member."
     )
+
+
+class CourseRunFieldBlock(blocks.FieldBlock):
+    """
+    Block class that allows selecting a course run
+    """
+
+    def get_courseruns(self):
+        """Lazy evaluation of the queryset"""
+        return (
+            apps.get_model("courses", "CourseRun")
+            .objects.live()
+            .values_list("courseware_id", "title")
+        )
+
+    def __init__(self, *args, required=True, help_text=None, **kwargs):
+        self.field = forms.ChoiceField(
+            choices=self.get_courseruns, help_text=help_text, required=required
+        )
+        super().__init__(*args, **kwargs)
+
+
+class CourseRunCertificateOverrides(blocks.StructBlock):
```

Comment:
Better to define in https://github.com/mitodl/mitxpro/blob/master/cms/blocks.py

Reply:
I don't get it, this is already in the same file :/

Reply:
Ok. I thought I am in models.py.